src/NuGet.Clients/VisualStudio/VisualStudio.csproj

```diff
@@ -103,13 +93,25 @@
     <None Include="NuGet.VisualStudio.nuspec" />
     <None Include="project.json" />
   </ItemGroup>
+  <Choose>
```

Comment:
Another candidate for just including the VisualStudio.Facade project. Future change.